include/roaring/portability.h

```diff
@@ -180,7 +180,7 @@ extern "C" {  // portability definitions are in global scope, not a namespace
 
 /* wrappers for Visual Studio built-ins that look like gcc built-ins __builtin_ctzll */
 /* result might be undefined when input_num is zero */
-static inline int roaring_trailing_zeroes(unsigned long long input_num) {
+inline int roaring_trailing_zeroes(unsigned long long input_num) {
```

Comment:
If we remove `static`, but leave it `inline`, we should add an `extern inline` to a `.c` file, or e.g. calling it directly in debug mode will fail with a linker error.